baselines/ippo/config/ippo_ff_puffer.yaml

```diff
@@ -2,36 +2,60 @@ mode: "train"
 use_rnn: false
 eval_model_path: null
 baseline: false
-data_dir: "data/processed/examples" # Dataset
+data_dir: "data/processed/validation"
 
 environment: # Overrides default environment configs (see pygpudrive/env/config.py)
   name: "gpudrive"
-  num_worlds: 50 # Number of parallel environments
-  k_unique_scenes: 1 # Number of unique scenes to sample from
+  num_worlds: 100 # Number of parallel environments
+  k_unique_scenes: 100 # Number of unique scenes to sample from
   max_controlled_agents: 32 # Maximum number of agents controlled by the model. Make sure this aligns with the variable kMaxAgentCount in src/consts.hpp
   ego_state: true
   road_map_obs: true
   partner_obs: true
   normalize_obs: true
-  reward_type: "sparse_on_goal_achieved"
+  remove_non_vehicles: true # If false, all agents are included (vehicles, pedestrians, cyclists)
+  use_lidar_obs: false # NOTE: Setting this to true currently turns of the other observation types
+  reward_type: "weighted_combination"
+  collision_weight: -0.005
+  off_road_weight: -0.005
```

Comment:
These are, I think too low relative to the discount. If you lose a step, discount causes you to lose 0.01 points. This is more than the collision penalty so as a consequence the agent will always take a collision to save a step. So I think you need to raise the values of these to be 2-3x higher

Reply:
You don't want to create a situation where an agent willingly accepts a collision to get to their goal sooner or recognizes that they can't get to their goal w/o a collision.